### DISCUSSION TITLE
User Registration on Livery

### DISCUSSION THREAD

**Discussion by GB014934** [flybywiresim/aircraft#4884] [General]
I have the latest development version - updated 09:00 (UK Time) 24/5/2021.

When I look at the external view of the aircraft - I am using the EasyJet livery - I see my user registration (G-TCEE) on the upper part of the fuselage as well as the one incluided in the livery. The one provided by the livery is in the correct position at the tail. 
I know I can remove my livery in the simulator but then it would be removed for all aircraft which I don't really want.
Is it possible for the livery to ignore the data in the user registration ?

Trevor

**Comment by ExampleWasTaken** [comment]
This is something that, if possible, the livery creator would have to implement.

**Reply by GB014934** [reply]
I have 4 liveries for the A320.  EasyJet, British Airways, FBW, and Airbus House.  The EasyJet and BAW liveries show both registrations - I guess the creator hard coded them in the correct place for their particular livery so my G-TCEE registration looks strange.  The FBW and Airbus House liveries do not have hard coded registrations included by the developer so my registration is the only one displayed.  

This is not that important - I may discontinue T-CEE for a while. I have seen it suggested on the FS2020 forums that it would be nice to have specific user registration per aircraft but I am not holding my breath for it to be implemented. 

My thanks for your response.
